CHANGELOG.md

```diff
@@ -1,3 +1,143 @@
+## 1.2.0
+This is a summary of the new features in version 1.2.0. For the full list, please refer to the [commits](https://github.com/serverpod/serverpod/commits/main/) on Github. Instructions for updating from 1.1 of Serverpod is available in our documentation [here](https://docs.serverpod.dev/upgrading/upgrade-to-one-point-two).
+
+### Main new features and fixes
+- feat: Adds official support for Windows.
+- feat: Adds Visual Studio Code extension.
+- feat: Syntax highlighting in model files.
+- feat: Adds LSP server for analyzing model files.
+- feat: CLI automatically detects modules withouth the need to modify the generator file.
+- feat: Validates project names on `serverpod create`.
+- feat: Validates Serverpod packages and CLI version in `serverpod generate`.
+- feat: Prompts user to update Serverpod when running an old version of the CLI.
+- feat: Improves exit codes for CLI.
+- feat: Improvements to output from CLI, including different formats for different platforms and run-modes.
+- feat: Progress animations in CLI.
+- feat: Uses CommandRunner for CLI.
+- feat: Adds global `--verbose` and `--quiet` flags to control log level.
+- feat: Developer version of `serverpod create` now creates a project referring to the local version of Serverpod.
+- feat: Adds `copyWith` methods on all generated model files.
+- feat: Makes it possible to call endpoint methods by specifying the method name in the path.
+- feat: Makes return headers configurable for API and OPTION HTTP calls.
+- feat: Adds `fromYaml` constructor to `ServerpodConfig`.
+- feat: Adds reference to all modules in config.
+- feat: Makes HTTP timeout configurable.
+- feat: Improves compatibility for `serverpod create` by not running Docker through tooling.
+- fix: Makes endpoint classes public to enable Dart doc.
+- fix: Serializable exceptions now work with modules.
+- fix: Handels invalid return types when parsing endpoint methods.
+- fix: Fixes localhost on Android emulator.
+- fix: Use explicit version for all Serverpod packages.
+- fix: Uses git version of CLI in local tests.
+- fix: Fixes typos in `serverpod create` start instructions.
+- fix: Makes include class fields private.
```

Comment:
This is a fix to something that we introduced in the relational system so it is not a change compared to 1.1.